app/routes/sign-in-oidc.js

```diff
@@ -25,11 +25,14 @@ module.exports = {
     }
   },
   handler: async (request, h) => {
-    const response = await getAccessToken(request.payload.code)
-    const state = decodeState(request.payload.state)
+    validateState(request, request.query.state)
+    const state = decodeState(request.query.state)
+    const response = await getAccessToken(request.query.code)
+    validateInitialisationVector(request, response.access_token)
```

Comment:
Need to validate initialisation vector after token has been retrieved as unlike the state it is not in the response with the auth code, it is part of the token itself.